pkg/network/protocols/testutil/tls_settings.go

```diff
@@ -3,6 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
+// Package testutil provides general utilities for protocols UTs.
```

Comment:
Is this comment actually needed in each file?

Reply:
golangci-lint is getting angry without it

Reply:
removed